addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -13,10 +15,21 @@ export class SuggestionService {
         this.env = env;
         this.orm = services.orm;
         this.store = services["mail.store"];
+        this.emojis;
     }
 
+    /**
+     * Returns list of supported delimiters, each supported
+     * delimiter is in an array [a, b, c] where:
+     * - a: chars to trigger
+     * - b: (optional) if set, the exact position in composer text input to allow using this delimiter
+     * - c: (optional) if set, this is the minimum amount of extra char after delimiter to allow using this delimiter
+     *
+     * @param {import('models').Thread} thread
+     * @returns {Array<[string, number, number]>}
+     */
     getSupportedDelimiters(thread) {
-        return [["@"], ["#"], ["::"]];
+        return [["@"], ["#"], ["::"], [":", undefined, 2]];
     }
```

Comment:
Data-structure of supported delimiters is not the best, I put a docstring to help code that relies on it to be more trusted.